packages/snap-preact/components/src/components/Organisms/Sidebar/Sidebar.tsx

```diff
@@ -69,8 +71,16 @@ export const Sidebar = observer((properties: SidebarProps): JSX.Element => {
 	return controller?.store?.loaded && controller?.store?.pagination?.totalResults > 0 && hasChildrenToRender ? (
 		<CacheProvider>
 			<div {...styling} className={classnames('ss__sidebar', className, { 'ss__sidebar--sticky': sticky })}>
+				{!hideTitleText ? (
+					<h4 className={classnames('ss__sidebar__title-text')} aria-atomic="true" aria-live="polite" {...mergedLang.titleText.all}>
```

Comment:
Lets align on a standard classname between Sidebar and MobileSidebar for the title element.

MobileSidebar has `ss__mobile-sidebar__header__title` - so maybe `ss__sidebar__title`?